setup.py

```diff
@@ -87,7 +87,7 @@ def run_tests(self):
         "six>=1.10.0,<2",
         "graphql-core>=2.1,<3",
         "graphql-relay>=0.4.5,<1",
-        "aniso8601>=3,<=6.0.*",
+        "aniso8601>=3,<=6.0",
```

Comment:
```suggestion
        "aniso8601>=3,<=6",
```
What do you think @AdrianLC ? That way it will accept minor versions too?

Reply:
Even better IMO, I only left it as 6.0 to not change the behavior intended on #935